package/MDAnalysis/analysis/hole2/hole.py

```diff
@@ -1303,7 +1303,7 @@ def plot_mean_profile(self, bins=100, range=None,
 
         binned, bins = self.bin_radii(frames=frames, bins=bins, range=range)
         mean = np.array(list(map(np.mean, binned)))
-        midpoints = 0.5 * bins[1:] + bins[:-1]
+        midpoints = 0.5 * bins[1:] + 0.5 * bins[:-1]
```

Comment:
```suggestion
        midpoints = 0.5 * (bins[1:] + bins[:-1])
```

Reply:
Just "commit the suggestion" in order to make the code a bit more concise.

Reply:
@miss77jun are you happy with @orbeckst's suggestion? If so, there's an easy way to add it to the code with the "Commit suggestion" button. If not, it would be good to explain why so we get what you're thinking!